docs/source/databricks-sql-warehouse.asciidoc

```diff
@@ -71,3 +71,16 @@ where `series name` = ?
 It is important to set `autocommit = true` since the Spark ODBC driver does not support transactions.
 
 Refer to https://learn.microsoft.com/en-us/azure/databricks/integrations/odbc/authentication[Authentication settings for the Databricks ODBC Driver] for alternative authentication options.
+
+Alternatively the `connection_string` can be split and passed as the individual entries:
+
+```
+[source.Databricks.connection]
+driver = ""
```

Comment:
I think we should default this to `/opt/simba/spark/lib/64/libsparkodbc_sb64.so` as used by the debian package.